kits/arm/ex_AR_kit_w_gripper.py

```diff
@@ -72,15 +72,31 @@
 # # Target variables
 # target_joints = np.zeros(arm.size)
 
+# Print Instructions
 instructions = (
     "Mode: {}\n"
     "A3: Gripper Control\n"
     "B1: Home\n"
     "B3: AR Control\n"
     "B6: Grav Comp\n"
     "B8: Quit")
-m.clear_text()
-m.add_text(instructions.format(run_mode))
+print(instructions.format(run_mode))
```

Comment:
It looks like the instructions were removed from device and only shown on the PC terminal.  Was this intentional?

Reply:
Ah -- nevermind.  Still sent below.  But now that these are printed to the terminal, the mode shown on the terminal will always be "softstart", whereas the app changes as the mode changes.